pkg/models/model_gallery.go

```diff
@@ -112,5 +112,42 @@ func (g *Gallery) listZipContents() ([]*zip.File, *zip.ReadCloser, error) {
 		return utils.NaturalCompare(a.Name, b.Name)
 	})
 
+	cover := contains(filteredFiles, "cover.jpg") // first image with cover.jpg in the name
+	if cover >= 0 {                               // will be moved to the start
+		reorderedFiles := reorder(filteredFiles, cover)
+		if reorderedFiles != nil {
+			return reorderedFiles, readCloser, nil
+		}
+	}
+
 	return filteredFiles, readCloser, nil
 }
+
+// return index of first occurenece of string x in name of zip contents, -1 otherwise
+func contains(a []*zip.File, x string) int {
+	for i, n := range a {
+		if strings.Contains(n.Name, x) {
+			logger.Debugf("Cover found in zip file %s.Position:%d", n.Name, i)
```

Comment:
This gets output a lot. Do we need it?

Reply:
I thought it was only printed when log level is set to debug. Not needed now though so removing.